e2e/specs/appointments.spec.ts

```diff
@@ -7,61 +7,138 @@ import { type Visit } from '@openmrs/esm-framework';
 let patient: Patient;
 let visit: Visit;
 
-test.beforeEach(async ({ api }) => {
+test.beforeAll(async ({ api }) => {
   patient = await generateRandomPatient(api);
   visit = await startVisit(api, patient.uuid);
 });
 
-test('Add an appointment', async ({ page, api }) => {
-  const appointmentsPage = new AppointmentsPage(page);
+test.describe.serial('Running appointment tests sequentially', () => {
+  test('Add an appointment', async ({ page }) => {
```

Comment:
@jayasanka-sack @kdaud I would like to seek your views concerning the use of `test.describe` in scenario of sequential operations e.g. `create, edit and delete`  like this one.
This is to improve some of our e2e tests going forward and prepare towards writing more advanced test cases.

Reply:
@chimanwadike the refactor should involve decoupling the test scenarios for adding, editing, and canceling an appointment.